commit 4fcaa66bbf69e63e14d84ce3157eb2b83c755c31
Author: Phạm Tiên Sinh <song.long.dai.duong@gmail.com>
Date:   Thu Jun 8 21:55:02 2023 +0800

    Revert "lahaina-v2-gpu: Improve GPU PowerLevel-Frequency table"
    
    This reverts commit 7cd11eabc0a97af1995a26a677b10b2ae79912a5.

diff --git a/arch/arm64/boot/dts/vendor/qcom/lahaina-v2-gpu.dtsi b/arch/arm64/boot/dts/vendor/qcom/lahaina-v2-gpu.dtsi
index 56d825450bf8..23b477a5fa54 100644
--- a/arch/arm64/boot/dts/vendor/qcom/lahaina-v2-gpu.dtsi
+++ b/arch/arm64/boot/dts/vendor/qcom/lahaina-v2-gpu.dtsi
@@ -36,12 +36,12 @@
 			#address-cells = <1>;
 			#size-cells = <0>;
 			qcom,speed-bin = <0>;
-			qcom,initial-pwrlevel = <5>;
+			qcom,initial-pwrlevel = <9>;
 
 			qcom,gpu-pwrlevel@0 {
 				reg = <0>;
-				qcom,gpu-freq = <820000000>;
-				qcom,level = <225>;
+				qcom,gpu-freq = <840000000>;
+				qcom,level = <RPMH_REGULATOR_LEVEL_TURBO_L1>;
 
 				qcom,bus-freq-ddr7 = <11>;
 				qcom,bus-min-ddr7 = <11>;
@@ -56,8 +56,8 @@
 
 			qcom,gpu-pwrlevel@1 {
 				reg = <1>;
-				qcom,gpu-freq = <780000000>;
-				qcom,level = <193>;
+				qcom,gpu-freq = <778000000>;
+				qcom,level = <RPMH_REGULATOR_LEVEL_TURBO>;
 
 				qcom,bus-freq-ddr7 = <11>;
 				qcom,bus-min-ddr7 = <11>;
@@ -72,8 +72,8 @@
 
 			qcom,gpu-pwrlevel@2 {
 				reg = <2>;
-				qcom,gpu-freq = <708000000>;
-				qcom,level = <145>;
+				qcom,gpu-freq = <738000000>;
+				qcom,level = <RPMH_REGULATOR_LEVEL_NOM_L1>;
 
 				qcom,bus-freq-ddr7 = <11>;
 				qcom,bus-min-ddr7 = <11>;
@@ -88,8 +88,8 @@
 
 			qcom,gpu-pwrlevel@3 {
 				reg = <3>;
-				qcom,gpu-freq = <638000000>;
-				qcom,level = <97>;
+				qcom,gpu-freq = <676000000>;
+				qcom,level = <RPMH_REGULATOR_LEVEL_NOM>;
 
 				qcom,bus-freq-ddr7 = <11>;
 				qcom,bus-min-ddr7 = <11>;
@@ -105,7 +105,7 @@
 			qcom,gpu-pwrlevel@4 {
 				reg = <4>;
 				qcom,gpu-freq = <608000000>;
-				qcom,level = <65>;
+				qcom,level = <RPMH_REGULATOR_LEVEL_SVS_L2>;
 
 				qcom,bus-freq-ddr7 = <8>;
 				qcom,bus-min-ddr7 = <7>;
@@ -120,8 +120,8 @@
 
 			qcom,gpu-pwrlevel@5 {
 				reg = <5>;
-				qcom,gpu-freq = <470000000>;
-				qcom,level = <1>;
+				qcom,gpu-freq = <540000000>;
+				qcom,level = <RPMH_REGULATOR_LEVEL_SVS_L1>;
 
 				qcom,bus-freq-ddr7 = <8>;
 				qcom,bus-min-ddr7 = <7>;
@@ -133,6 +133,71 @@
 
 				qcom,acd-level = <0xa82c5ffd>;
 			};
+
+			qcom,gpu-pwrlevel@6 {
+				reg = <6>;
+				qcom,gpu-freq = <491000000>;
+				qcom,level = <RPMH_REGULATOR_LEVEL_SVS_L0>;
+
+				qcom,bus-freq-ddr7 = <8>;
+				qcom,bus-min-ddr7 = <6>;
+				qcom,bus-max-ddr7 = <10>;
+
+				qcom,bus-freq-ddr8 = <7>;
+				qcom,bus-min-ddr8 = <5>;
+				qcom,bus-max-ddr8 = <7>;
+
+				qcom,acd-level = <0xa82c5ffd>;
+			};
+
+			qcom,gpu-pwrlevel@7 {
+				reg = <7>;
+				qcom,gpu-freq = <443000000>;
+				qcom,level = <RPMH_REGULATOR_LEVEL_SVS>;
+
+				qcom,bus-freq-ddr7 = <8>;
+				qcom,bus-min-ddr7 = <6>;
+				qcom,bus-max-ddr7 = <10>;
+
+				qcom,bus-freq-ddr8 = <7>;
+				qcom,bus-min-ddr8 = <5>;
+				qcom,bus-max-ddr8 = <7>;
+
+				qcom,acd-level = <0xa82d5ffd>;
+			};
+
+			qcom,gpu-pwrlevel@8 {
+				reg = <8>;
+				qcom,gpu-freq = <379000000>;
+				qcom,level = <RPMH_REGULATOR_LEVEL_LOW_SVS_L1>;
+
+				qcom,bus-freq-ddr7 = <8>;
+				qcom,bus-min-ddr7 = <6>;
+				qcom,bus-max-ddr7 = <10>;
+
+				qcom,bus-freq-ddr8 = <6>;
+				qcom,bus-min-ddr8 = <5>;
+				qcom,bus-max-ddr8 = <7>;
+
+				qcom,acd-level = <0x882e5ffd>;
+			};
+
+			qcom,gpu-pwrlevel@9 {
+				reg = <9>;
+				qcom,gpu-freq = <315000000>;
+				qcom,level = <RPMH_REGULATOR_LEVEL_LOW_SVS>;
+
+				qcom,bus-freq-ddr7 = <2>;
+				qcom,bus-min-ddr7 = <2>;
+				qcom,bus-max-ddr7 = <9>;
+
+				qcom,bus-freq-ddr8 = <2>;
+				qcom,bus-min-ddr8 = <2>;
+				qcom,bus-max-ddr8 = <7>;
+
+				qcom,acd-level = <0x882f5ffd>;
+			};
+
 		};
 
 		qcom,gpu-pwrlevels-1 {
